2024/01/19 18:34:19 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "helpme". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('helpme')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('helpme')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('site/helpme')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/helpme
---
2024/01/19 18:38:08 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "feature". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('feature')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('feature')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('home/feature')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/home/feature
---
2024/01/19 18:47:31 [error] [exception.ParseError] ParseError: syntax error, unexpected '<', expecting end of file in C:\xampp1\htdocs\YII_Framework\protected\views\dashboard\view.php:4
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp1\\htdoc...', NULL, true)
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp1\\htdoc...', NULL, true)
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(785): CController->renderPartial('view', NULL, true)
#3 C:\xampp1\htdocs\YII_Framework\protected\controllers\DashboardController.php(10): CController->render('view')
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\actions\CInlineAction.php(49): DashboardController->actionHome()
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('home')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard/home')
#10 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/dashboard/home
---
2024/01/25 19:30:44 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "favicon.ico". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
HTTP_REFERER=http://dev.yiiapp.com/
---
2024/01/25 19:54:05 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "home". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('home')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('home')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('site/home')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/home
---
2024/01/25 19:55:47 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "gii". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('gii')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/gii
---
2024/01/26 17:24:51 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "pegawai". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('pegawai')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('pegawai')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('site/pegawai')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/pegawai
---
2024/01/26 17:26:37 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "_view". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('_view')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('_view')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/_view')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/pegawai/_view
HTTP_REFERER=http://dev.yiiapp.com/index.php/site/contact
---
2024/01/26 17:26:51 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "_view". in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php:486
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(270): CController->missingAction('_view')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('_view')
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/_view')
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/pegawai/_view
HTTP_REFERER=http://dev.yiiapp.com/index.php/site/contact
---
2024/01/26 17:32:42 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/admin')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/pegawai/admin
HTTP_REFERER=http://dev.yiiapp.com/index.php/pegawai/index
---
2024/01/26 18:10:36 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/admin')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/pegawai/admin
HTTP_REFERER=http://dev.yiiapp.com/index.php/pegawai/index
---
2024/01/26 18:10:52 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('delete')
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/delete')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/pegawai/delete/3
HTTP_REFERER=http://dev.yiiapp.com/index.php/pegawai/3
---
2024/01/26 18:11:23 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/admin')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/pegawai/admin
HTTP_REFERER=http://dev.yiiapp.com/index.php/pegawai/index
---
2024/01/26 18:11:28 [error] [exception.CHttpException.403] CHttpException: You are not authorized to perform this action. in C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php:183
Stack trace:
#0 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#3 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\web\CWebApplication.php(141): CWebApplication->runController('pegawai/admin')
#9 C:\xampp1\htdocs\YII Framework\YII-1.1\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 C:\xampp1\htdocs\YII_Framework\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/pegawai/admin
HTTP_REFERER=http://dev.yiiapp.com/index.php/pegawai/3
---
